docs/book/src/topics/rosa/creating-a-cluster.md

```diff
@@ -55,8 +55,9 @@ Once Step 3 is done, you will be ready to proceed with creating a ROSA cluster u
     ```shell
     clusterctl generate cluster <cluster-name> --from templates/cluster-template-rosa.yaml > rosa-capi-cluster.yaml
     ```
+  Note: The AWS rule name must be no more than 64 characters in length. Otherwise an error will be returned. Truncate values exceeding 64 characters.
```

Comment:
```suggestion
  Note: The AWS role name must be no more than 64 characters in length. Otherwise an error will be returned. Truncate values exceeding 64 characters.
```

Reply:
updated